scripts/data/wikidata_oceans.csv

```diff
@@ -0,0 +1,40 @@
+Name,ID
+pan-african ocean,Q1026852
+western interior seaway,Q1083697
+north pacific,Q12353254
+north pacific ocean,Q12353254
+north pacific basin,Q12353254
+eastern ocean,Q1239
+eastern sea,Q1239
+indian ocean,Q1239
+south pacific ocean,Q12355425
+paleo-tethys ocean,Q1432114
+khanty ocean,Q1740563
+rheic ocean,Q1891116
+mirovia,Q1938548
+cretaceous sea,Q19760534
+tethys ocean,Q208136
+tethys,Q208136
+tethys sea,Q208136
+neotethys,Q208136
+panthalassa,Q208378
+proto-tethys ocean,Q2113702
+ural ocean,Q2233441
+turgai sea,Q2274793
+paratethys,Q288291
+central american seaway,Q3202980
+sundance sea,Q3310378
+southern ocean,Q7354
+antarctic ocean,Q7354
+valais ocean,Q763928
+northern ocean,Q788
+arctic ocean,Q788
+arctic sea,Q788
+tornquist sea,Q7826151
+atlantic,Q97
+atlantic ocean,Q97
+great western ocean,Q97
+atlantic sea,Q97
+pacific ocean,Q98
+pacific sea,Q98
+pacific,Q98
```

Comment:
Similarly here also we can remove the qualifiers like "ocean" or "sea" from the entries.

Reply:
Can I directly strip it in the csv or I have to do it in the download_whatever.py file?

Reply:
The download scripts should download exactly what the wikidata query returns. `scripts/util.py` is responsible for converting all csvs into docbins. This is taken care in that script. https://github.com/gwf-uwaterloo/waterwheel/blob/master/scripts/util.py#L78 strips away some of the words. You can add new words here and check if the results are correct.